sablo/META-INF/resources/sablo/js/websocket.ts

```diff
@@ -404,6 +409,8 @@ webSocketModule.factory('$webSocket',
 				sendMessageObject(response);
 			}
 		} finally {
+			currentRequestInfo = undefined;
```

Comment:
Remove this line; it can be done below using oldRequestInfo so
    currentRequestInfo  = oldRequestInfo;
right above the optimizeAndCallFormScopeDigest call below.

Reasoning is that currentRequestInfo I think should still be set when executing the afterIncommingMessageWasHanded functions + oldRequestInfovalue is needed due to situations similar to what the comment describes where oldFTEAIMWH is declared.

Reply:
There is an strange scenario in Firefox. Where, when you debug or when the client side code that executes when the msg. from server is being processed ends up calling some blocking api of the browser like window.alert(...) - and then another message comes in from the server through the websocket, it will execute on top of the 'blocked' stack. Which means this code that handles server message can run on top of itself (nested) and then weird errors occur.
I agree that normally - in the js world it should not execute like that but one after the other and then there would be no need for that old... and restore.
I see @gboros  merged this in already so I will make this small adjustment in code.